core/src/main/java/tachyon/master/MasterInfo.java

```diff
@@ -2132,6 +2174,7 @@ public boolean rename(TachyonURI srcPath, TachyonURI dstPath) throws FileDoesNot
     synchronized (mRootLock) {
       Inode inode = getInode(srcPath);
       if (inode == null) {
+        mMasterSource.incRenameOps();
```

Comment:
Why this counter is incremented within the error case? If this is by purpose, better put some comments as it is not obvious to me.